CITATION.cff

```diff
@@ -0,0 +1,81 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Frictionless Python Framework
+message: >-
+  To cite the Frictionless Python Framework in publications
+  please use:
+type: software
+authors:
+  - given-names: Evgeny
+    family-names: Karev
+    email: eskarev@gmail.com
+  - given-names: Pierre
+    family-names: Camilleri
+    affiliation: multi.coop
+    email: pierre.camilleri@multi.coop
+  - given-names: Vitor
+    family-names: Baptista
+    affiliation: Fiquem Sabendo
+  - given-names: Georgiana
+    family-names: Bere
+  - given-names: Andrea
+    family-names: Borruso
+    affiliation: OnData
+  - given-names: Peter
+    family-names: Desmet
+    orcid: 'https://orcid.org/0000-0002-8442-8025'
+    affiliation: INBO
+  - given-names: Shashi
+    family-names: Gharti
+    affiliation: Robust IT Concepts
+  - given-names: Augusto
+    family-names: Herrmann
+    affiliation: >-
+      Ministry of Management and Innovation in Public
+      Services in Brazil
+  - given-names: Adam
+    family-names: Kariv
+    affiliation: While True Industries
+  - given-names: Chris
+    family-names: Shaw
+    affiliation: Democracy Club
+  - given-names: Paul
+    family-names: Walsh
+    affiliation: LinkDigital
+  - given-names: Lilly
+    family-names: Winfree
+    affiliation: Anaconda, Inc.
+    orcid: 'https://orcid.org/0000-0001-7120-8536'
+  - given-names: Edgar
+    family-names: Zanella Alvarenga
+    affiliation: Digi Sapiens
+  - given-names: Jesper
+    family-names: Zedlitz
+    orcid: 'https://orcid.org/0000-0003-2664-5010'
+  - name: Open Knowledge Foundation
+    city: London
+    country: GB
+  - given-names: Sara
+    family-names: Petti
+    affiliation: Open Knowledge Foundation
+    email: sara.petti@okfn.org
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.4663759
```

Comment:
This is the DOI for version 5.18 specifically. It is fine to use that one if you're going to update this every time a new version is created (cf. just like you have to update the `version` and `date-released`). If don't want to do that, refer to the versionless DOI https://doi.org/10.5281/zenodo.4663759.

Note: I would refer to versionless DOI in README.